Tests not passing

`/src/mui/field/DateField.spec.js#18`, `/src/mui/field/DateField.spec.js#28` and `/src/mui/field/DateField.spec.js#38` the test may fail because of the timezone. By default a date is set on `00:00:00`. My timezone is `UTC-3` so on the DateField tests the time shows the wrong date (previous day at `21:00:00`). 

Also it asked for `aor-json-rest-client`, `aor-language-french` and `aor-rich-text-input` when running the tests. Is something missing on `package.json`?

```
  1) <DateField /> should render a date:

      AssertionError: '<span>4/22/2017</span>' == '<span>4/23/2017</span>'
      + expected - actual

      -<span>4/22/2017</span>
      +<span>4/23/2017</span>

      at Context.<anonymous> (src/mui/field/DateField.spec.js:17:45)

  2) <DateField /> should pass the options prop to Intl.DateTimeFormat:

      AssertionError: '<span>Saturday, April 22, 2017</span>' == '<span>Sunday, April 23, 2017</span>'
      + expected - actual

      -<span>Saturday, April 22, 2017</span>
      +<span>Sunday, April 23, 2017</span>

      at Context.<anonymous> (src/mui/field/DateField.spec.js:27:76)

  3) <DateField /> should use the locales props as an argument to Intl.DateTimeFormat:

      AssertionError: '<span>22/04/2017</span>' == '<span>23/04/2017</span>'
      + expected - actual

      -<span>22/04/2017</span>
      +<span>23/04/2017</span>

      at Context.<anonymous> (src/mui/field/DateField.spec.js:37:91)

```


and then I added on line 18 the following:

```
      const date = new Date('2017-04-23');
      console.log(date);
      console.log(date.toLocaleString('en-US', {}));
      console.log(date.toLocaleString());
      console.log(date.toLocaleDateString('en-US', {}));
      console.log(date.toLocaleDateString());
```

and got this output:

```
2017-04-23T00:00:00.000Z
4/22/2017, 9:00:00 PM
4/22/2017, 9:00:00 PM
4/22/2017
4/22/2017
```

Now. if I set the hours:
```

      const date = new Date('2017-04-23 13:00:00');
      console.log(date);
      console.log(date.toLocaleString('en-US', {}));
      console.log(date.toLocaleString());
      console.log(date.toLocaleDateString('en-US', {}));
      console.log(date.toLocaleDateString());
````
then
```
2017-04-23T16:00:00.000Z
4/23/2017, 1:00:00 PM
4/23/2017, 1:00:00 PM
4/23/2017
4/23/2017
```

--- a/src/mui/field/DateField.spec.js
+++ b/src/mui/field/DateField.spec.js
@@ -16,36 +16,42 @@ describe('<DateField />', () => {
 
     it('should render a date', () => assert.equal(
         shallow(<DateField record={{ foo: new Date('2017-04-23') }} source="foo" locales="en-US" />).html(),
-        '<span>4/23/2017</span>',
+        `<span>${new Date('2017-04-23').toLocaleDateString('en-US')}</span>`,
     ));
 
     it('should render a date and time when the showtime prop is passed', () => assert.equal(
         shallow(<DateField record={{ foo: new Date('2017-04-23 23:05') }} showTime source="foo" locales="en-US" />).html(),
-        '<span>4/23/2017, 11:05:00 PM</span>',
+        `<span>${new Date('2017-04-23 23:05').toLocaleString('en-US')}</span>`,
     ));
 
-    it('should pass the options prop to Intl.DateTimeFormat', () => assert.equal(
-        shallow(<DateField record={{ foo: new Date('2017-04-23') }} source="foo" locales="en-US" options={{
+    it('should pass the options prop to toLocaleString', () => {
+        const date = new Date('2017-04-23');
+        const options = {
             weekday: 'long',
             year: 'numeric',
             month: 'long',
             day: 'numeric',
-        }} />).html(),
-        '<span>Sunday, April 23, 2017</span>',
-    ));
+        };
+        return assert.equal(
+            shallow(<DateField
+                record={{ foo: date }} source="foo" locales="en-US" options={options}
+            />).html(),
+            `<span>${date.toLocaleDateString('en-US', options)}</span>`,
+        );
+    });
 
-    it('should use the locales props as an argument to Intl.DateTimeFormat', () => assert.equal(
+    it('should use the locales props as an argument to toLocaleString', () => assert.equal(
         shallow(<DateField record={{ foo: new Date('2017-04-23') }} source="foo" locales="fr-FR" />).html(),
-        '<span>23/04/2017</span>',
+        `<span>${new Date('2017-04-23').toLocaleDateString('fr-FR')}</span>`,
     ));
 
     it('should use custom styles passed as an elStyle prop', () => assert.equal(
         shallow(<DateField record={{ foo: new Date('01/01/2016') }} source="foo" locales="en-US" elStyle={{ margin: 1 }} />).html(),
-        '<span style="margin:1px;">1/1/2016</span>',
+        `<span style="margin:1px;">${new Date('1/1/2016').toLocaleDateString('en-US')}</span>`,
     ));
 
     it('should handle deep fields', () => assert.equal(
         shallow(<DateField record={{ foo: { bar: new Date('01/01/2016') } }} source="foo.bar" locales="en-US" />).html(),
-        '<span>1/1/2016</span>',
+        `<span>${new Date('1/1/2016').toLocaleDateString('en-US')}</span>`,
     ));
 });

